components/webgpu/lib.rs

```diff
@@ -467,23 +475,64 @@ impl<'a> WGPU<'a> {
                         let descriptor = BindGroupDescriptor {
                             label: None,
                             layout: bind_group_layout_id,
-                            bindings: bindings.as_slice(),
+                            entries: bindings.as_slice(),
                         };
-                        let _ = gfx_select!(bind_group_id =>
+                        let result = gfx_select!(bind_group_id =>
                             global.device_create_bind_group(device_id, &descriptor, bind_group_id));
+                        if let Some(s_id) = scope_id {
+                            let &pipeline_id = self.devices.get(&WebGPUDevice(device_id)).unwrap();
+                            let op_result = if let Err(e) = result {
```

Comment:
this code gets repeated, can we move it to a function

Reply:
`result` is of a different type for each method so we'll have to assign `op_result` separately at each instance but the `send(WebGPUOpResult)` can be extracted.

Reply:
this code doesn't particularly care *what* type `result` has, as long as it's `Result<T: Display, ()>`, so a function can still help here